_tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: default
+title: Remote Store Stats API 
+nav_order: 20
+parent: Remote-backed storage
+grand_parent: Availability and recovery
+---
+
+# Remote Store Stats API
+Introduced 2.8
+{: .label .label-purple }
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/1968).
+{: .warning}
+
+Use the Remote Store Stats API to monitor shard-level remote store performance.
+
+## Path and HTTP methods
+
+```json
+GET _remotestore/stats/<index_name>
+GET _remotestore/stats/<index_name>/<shard_id>
+```
+
+## Path parameters
+
+The following table lists the available path parameters. All path parameters are optional.
+
+Parameter | Type | Description
+:--- | :--- | :---
+`index_name` | String | The index name.
```

Comment:
@linuxpi - index name can be an index pattern too, right?

Reply:
How about a comma-separated list, does it take that as well?

Reply:
@linuxpi ^

Reply:
This only support single index name as of now. So you cannot specify multiple indices via comma separated list or index name pattern

Reply:
Checked again and index patterns  works. Thanks @ashking94 for pointed out.

So something like `my-index-*` would work and match indexes like `my-index-1` , `my-index-2`